packages/ai/src/ai/web/endpoints/webhook.py

```
@@ -0,0 +1,195 @@
+                token=token,
+                pipeline_id=pipeline_id,
+                status='accepted',
+                created_at=now,
+            )
+
+            return response(data=webhook_resp.model_dump(mode='json'))
+        finally:
+            async with _running_lock:
+                _running_count -= 1
+
+    except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)